tests/test_speedcopy.py

```diff
@@ -13,18 +14,36 @@ def teadown_function(function):
     speedcopy.unpatch_copyfile()
 
 
-def test_copy(tmpdir):
+def test_copy_abs(tmpdir):
     src = tmpdir.join("source")
     dst = tmpdir.join("destination")
     with open(str(src), "wb") as f:
-        f.write(os.urandom(5 * 1024 * 1024))
+        f.write(os.urandom(_FILE_SIZE))
     f.close()
 
     shutil.copyfile(str(src), str(dst))
 
     assert os.path.isfile(str(dst))
 
 
+def test_copy_rel(tmpdir):
+    cwd = os.getcwd()
+    os.chdir(tmpdir)
```

Comment:
this has to be `os.chdir(str(tmpdir)` as tmpdir in pytest is not str but **LocalPath**

Reply:
OK, done.

Reply:
Looks like there was a genuine test failure on the Linux side.  I don't really have the time to dive into that side of things I'm afraid, I have pushed a speculative fix but I have no access to a Linux machine to test.

Reply:
yep, I am on it. It seems that on linux side it doesn't find `tmpdir` for destination and therefor failing while trying to query filesystem type.